Tests/MongoSwiftTests/TestUtils.swift

```diff
@@ -4,6 +4,70 @@ import mongoc
 import Nimble
 import XCTest
 
+typealias DeleteOneModel = MongoCollection<Document>.DeleteOneModel
+typealias DeleteManyModel = MongoCollection<Document>.DeleteManyModel
+typealias InsertOneModel = MongoCollection<Document>.InsertOneModel
+typealias ReplaceOneModel = MongoCollection<Document>.ReplaceOneModel
+typealias UpdateOneModel = MongoCollection<Document>.UpdateOneModel
+typealias UpdateManyModel = MongoCollection<Document>.UpdateManyModel
+
+/// A struct representing a server version.
+internal struct ServerVersion: Comparable, Decodable {
```

Comment:
This was moved out of a `MongoClient` extension so that it could be used on its own inside the spec test runner. I also added `Comparable` conformance so that we can use the comparison operators instead of the english versions.